try_obsconfig_model.ipynb

```diff
@@ -0,0 +1,622 @@
+{
+ "cells": [
```

Comment:
Accidental commit?

Reply:
strange - I don't know how that got committed...